Yubico.YubiKey/src/Yubico/YubiKey/Piv/Commands/VerifyUvResponse.cs

```diff
@@ -39,15 +39,15 @@ namespace Yubico.YubiKey.Piv.Commands
     ///
     /// <item>
     /// <term><see cref="ResponseStatus.Success"/></term>
-    /// <description>The biometric authentication succeeded. GetData returns temporary pin if requested.
+    /// <description>The biometric authentication succeeded. GetData returns the temporary pin if requested.
     /// </description>
     /// </item>
     ///
     /// <item>
     /// <term><see cref="ResponseStatus.AuthenticationRequired"/></term>
     /// <description>The biometric authentication failed. GetData returns null. <see cref="AttemptsRemaining"/> 
     /// returns the number of retries remaining. If the number of retries is 0, the biometric authentication is 
-    /// blocked and the client should use PIN authentication <see cref="VerifyPinCommand"/>.</description>
+    /// blocked and the client should authenticate with the PIV PIN via <see cref="VerifyPinCommand"/>.</description>
```

Comment:
Assuming this was referring to the PIV PIN, but please correct if wrong.

Reply:
correct 👍🏻